TestProject/BadlyFormattedTests.cs.expected

```diff
@@ -0,0 +1,57 @@
+﻿//  Copyright (c) rubicon IT GmbH
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using NUnit.Framework;
+using TestProject.TestClasses;
+using TestProject.TestInterfaces;
+using Rhino.Mocks;
+using Moq;
+using MockRepository = Rhino.Mocks.MockRepository;
+
+namespace TestProject
+{
+[TestFixture]
+public class BadlyFormattedTests
+{
+  private readonly TestClass _memberCaller=new TestClass();
+  private Mock<ITestInterface> _mock = new Mock<ITestInterface>();
+
+          [SetUp]
+  public void SetUp ()
+  {
+    var mock = new Mock<ITestInterface>();
+
+    Console   .   WriteLine ("Test")         ;
+  }
+
+          public void AnyTest ()
+          {
+Page namingContainer = new Page();
+            var parentControlMock = new Mock<IControl>() { CallBase = true };
+            var childControlMock    =    new Mock<IControl>() { CallBase = true };
+            var sequence1 = new MockSequence();
+            childControlMock.InSequence (sequence1).Setup (mock => PrivateInvoke.InvokeNonPublicMethod (mock, "OnInit", EventArgs.Empty)).Verifiable();
```

Comment:
Moq has a dedicated `Protected()` method, which means we could write 
childControlMock.InSequence (sequence1).Protected().Setup ("OnInit", true, EventArgs.Empty).Verifiable();`

For some reason, this wasn't considered during the original evaluation of how to migrate to Moq and thus was not part of the automatic rewrite. I think we should be able to just do a structural refactoring for this after the fact, but please verify that the test actually works when using PrivateInvoke to call the protected members (failes if the method isn't called, sets up a return value, etc). I'm also interested in learing how the diagnostic output will handle PrivateInvoke.

Reply:
The test does not work with the PrivateInvoke method, also .InSequence() seems to be incompatible with .Protected()

Reply:
Okay, just checked, in TypePipe we already use `Protected()` but not in combination with Sequence. So, the change to `Protected()` is probably somehting that should be done during the rewrite. 

Question remains what we do with the sequence. There's a `SetupSequence` method but we decided to use the `InSequence` api because it's easier to rewrite and allows us to use different mock-objects. 

Question: if we simply keep the rewrite to use `InSequence` with `Protected`, will the tests fail during the setup and/or the execution with a meaningful error message our would the problem remain hidden or otherwise be difficult to diagnose. Depending on the answer, we can decide what we do (e.g. introduce a non-existing method (e.g. `InSequenceProtected`) so we get a compiler error that tells us what the problem is.

Reply:
Just checked: the fluent API just doesn't provide the combination of `InSequence` and `Protected` so we could just rewrite to this regardless of this being allowed and then manually fix it afterwards. There probably shouldn't be that many occurances of a sequential non-public all anyway.

Reply:
with `.InSequence().Protected()` you would already get a compiler error

if `.Protected()` is missing the test would throw an exception with the following message `System.NotSupportedException : Unsupported expression: mock => PrivateInvoke.InvokeNonPublicMethod(mock, "OnInit", new[] { EventArgs.Empty })
Static members (here: PrivateInvoke.InvokeNonPublicMethod) may not be used in setup / verification expressions.`

Reply:
okay, then we go with `.InSequence().Protected()`, that way, you get the most automatic conversion out of it.